### DISCUSSION TITLE
Iron Shulkers, Raw Modded Ores and Bright Potion

### DISCUSSION THREAD

**Discussion by BL00DKNIGHT** [Roadhog360/Et-Futurum-Requiem#163] [Ideas]
The title explains all.
also amazing work that you guys have making this mod!

**Comment by Roadhog360** [comment]
The first two are already in the works. What is the "bright" potion? Are you referring to the glowing outline effect? I don't believe the glowing effect is actually possible in 1.7.10.

**Comment by GregoriusT** [comment]
Okay just chiming in, I am pretty sure you could probably implement something like that outline by hacking and glitching some GL-States MAYBE, but I am not good enough at that to really do it well...

At the very least MC is inefficient enough to not cull any Mobs behind Walls so you can definitely just change their rendering somehow.

It might be laggy to even hook into that though...

**Comment by Roadhog360** [comment]
I don't know how to do that but I'm sure that it was only possible due to rendering changes in 1.8 and 1.9. We would also need to find a way to cancel culling of mobs when the player has optimization mods installed.

**Comment by BL00DKNIGHT** [comment]
on the bright potion it's like the terraria one, makes the player emit light like if it was holding an torch with dynamic lights on
